### DISCUSSION TITLE
Allow increasing volume above default

### DISCUSSION THREAD

**Discussion by Aran-Fey** [File-New-Project/EarTrumpet#1227] [Ideas]
The title is a little vague, but that's because there are two different ways to implement this feature I want. Let me provide some background info to make this easier to explain.

The problem I have is that most "local" audio (games, Windows notification sounds, etc) is much louder than audio on the internet (YouTube, Twitch, etc). The obvious solution is, of course, to make my browser's volume louder than everything else.

Unfortunately, that's not as easy as it sounds. Currently, the only way to do that is to increase the overall (speaker) volume to make *everything* louder, and then reduce the volume of every process except for my browser. The problem with this is that every new process (for example, when I install a new game) starts at maximum volume and blows my ears out.

So what's the feature request? A way to make one process louder than all others. I can think of two ways to implement this:

1. Make it possible to increase the volume of a process above 100.
2. Make it possible to change the default volume from 100 to something else, so that new processes don't start at maximum volume.

**Comment by G3iMacMan** [comment]
Agreed. This is what I'm looking for too. I used a web browser plug-in called "Volume Master" to go above 100% on web pages but I'm looking for an OS-level solution. The issue is, I used a TV/Media Display for my main PC monitor and speakers. When I get the TV volume set appropriately on the PC, and switch to the Monitor's Media Apps, the volume is WAY too loud. I'd appreciate having a plugin on the PC that would bring the line volume more in line with the rest of the media volume played on the display.

**Reply by YoDawgYo** [reply]
Set you game volume to 100% and then everything else to 50%. Problem solved.  Going above 100% can cause problems for your devices and sound quality.

**Reply by G3iMacMan** [reply]
Here is the issue... My monitor is also a TV/Media center. The volumen coming out of my PC with all things at max is far lower then any other media input. So, if I have the volume set for the PC at a comfortable level, the next input is far far too loud.

**Reply by YoDawgYo** [reply]
> Here is the issue... My monitor is also a TV/Media center. The volumen coming out of my PC with all things at max is far lower then any other media input. So, if I have the volume set for the PC at a comfortable level, the next input is far far too loud.

This sounds to me like a possible driver and/or hardware issue. 
Tell me more about your TV/Media center and other hardware you are using. 
Also check this out and beware, boosting past 100% can and likely will cause damage to your audio chips and anything connected to it.
https://www.pcworld.com/article/2647501/windows-100-volume-is-a-phony-limit-heres-how-i-boost-audio-6x-higher.html

**Comment by tendvbs** [comment]
I understand. I understand very much.
I totally agree.

I've been looking for an app that can do this, but I can't find one.
It would be great if EarTrumpet could do it.

It would be nice if we could specify the absolute volume for each app individually.
Not “as a percentage of the maximum system volume” as it is now.


The Windows volume management is wrong to begin with.
“Basically maximum.”
“It is possible to lower the volume for each app.”
“It is impossible to increase the volume for each app. In that case, turn up the overall volume.”

**Comment by OneGiantChicken** [comment]
I would find these 2 features greatly useful

**Comment by wancor1** [comment]
I need this

**Reply by YoDawgYo** [reply]
What I think you really need is a SCHIIT DAC and Amplifier.
